CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 Versions
 ========
+2023-09-20
+----------
+* Python : build arm64 docker image
```

Comment:
could you move this line to the "2023-09-30" section, as we aim to include this PR in the monthly release ?
Thanks !